src/main/java/com/marek/onlinebookstore/dto/order/OrderDto.java

```diff
@@ -0,0 +1,21 @@
+package com.marek.onlinebookstore.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+public record OrderDto(
```

Comment:
Consider using class instead record here for the future changes where you can add additional logic here.